### ISSUE TITLE
bug report: maybe miss makezero in 

### ISSUE THREAD

**Issue by alingse** [montanaflynn/stats#84]
I was running github actions to run linter [makezero](https://github.com/ashanbrown/makezero) for top github golang repos.

see issues https://github.com/alingse/go-linter-runner/issues/1

and the github actions output https://github.com/alingse/go-linter-runner/actions/runs/9243004504/job/25426564828
```
====================================================================================================
append to slice `mode` with non-zero initialized length at https://github.com/montanaflynn/stats/blob/master/mode.go#L23:11
append to slice `mode` with non-zero initialized length at https://github.com/montanaflynn/stats/blob/master/mode.go#L34:[10](https://github.com/alingse/go-linter-runner/actions/runs/9243004504/job/25426564828#step:4:11)
====================================================================================================
```

https://github.com/montanaflynn/stats/blob/master/mode.go#L16C1-L23
```go
	mode = make([]float64, 5)
	cnt, maxCnt := 1, 1
	for i := 1; i < l; i++ {
		switch {
		case c[i] == c[i-1]:
			cnt++
		case cnt == maxCnt && maxCnt != 1:
			mode = append(mode, c[i-1])
```

maybe the `mode = make([]float64, 5)` should be `mode = make([]float64, 0, 5)`

**Comment by ccoVeille** [comment]
> I was running github actions to run linter [makezero](https://github.com/ashanbrown/makezero) for top github golang repos.

Hi

I'm interested in the methodology you used to perform such a test. Do you have a public repository for this ? Or did you use a tool.

I'm sorry, if I'm hijacking the discussion.

Your issue is important and relevant, but I'm curious because I have a similar need

**Comment by ccoVeille** [comment]
I feel stupid. That's the whole purpose of your repository

https://github.com/alingse/go-linter-runner/issues/1

**Comment by alingse** [comment]
no, I will write more linter's yaml config like this https://github.com/alingse/go-linter-runner/blob/main/jobs/alingse-makezero.yaml#L4

I has some idea about linter https://github.com/alingse/sundrylint/issues/2  , I want to check the idea in real-world 

and than add the useful one to golangci-lint 🤔

and I want to add some auto-report-issuse to process the github actions output.
